basebinaries/libjailbreak/libjailbreak.h

```diff
@@ -0,0 +1,18 @@
+//
+//  libjailbreak.h
+//  libjailbreak
+//
+//  Created by Jamie Bishop on 29/01/2018.
+//  Copyright © 2018 Electra Team. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+struct __attribute__((__packed__)) JAILBREAKD_ENTITLE_PID {
```

Comment:
none of the public functions take this struct, it can be moved into the .m to reduce clutter

Reply:
They *were* going to until I changed at the last minute